manifests/bridge/debian.pp

```diff
@@ -0,0 +1,56 @@
+# = Define: network::bridge::debian
+#
+define network::bridge::debian(
+  $ports,
+  $ensure    = present,
+  $ipaddress = undef,
+  $netmask   = undef,
+  $method    = undef,
+  $family    = undef,
+  $onboot    = undef,
+  $mtu       = undef,
+
+  $ageing     = undef,
+  $bridgeprio = undef,
+  $fd         = undef,
+  $gcint      = undef,
+  $hello      = undef,
+  $hw         = undef,
+  $maxage     = undef,
+  $maxwait    = undef,
+  $pathcost   = undef,
+  $portprio   = undef,
+  $stp        = undef,
+  $waitport   = undef,  
+) {
+
+  $raw = {
+      'bridge_ports'      => join($ports, ' '),
+      'bridge_ageing'     => $ageing,
+      'bridge_bridgeprio' => $bridgeprio,
+      'bridge_fd'         => $fd,
+      'bridge_gcint'      => $gcint,
+      'bridge_hello'      => $hello,
+      'bridge_hw'         => $hw,
+      'bridge_maxage'     => $maxage,
+      'bridge_maxwait'    => $maxwait,
+      'bridge_pathcost'   => $pathcost,
+      'bridge_portprio'   => $portprio,
+      'bridge_stp'        => $stp,
+      'bridge_waitport'   => $waitport,
+  }
+
+  $opts = compact_hash($raw)
```

Comment:
This function was written by @adrienthebo (as per https://github.com/puppet-community/puppet-network/blob/98a0275f52ed54e41544fcfad67dd6381ae1ddf1/lib/puppet/parser/functions/compact_hash.rb) and it was used at the time I created the patch in other part of this module (which I used as a base). Anyway since my PR won't be merged, I don't think it deserves time and efforts to port this code.